mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -52,6 +54,18 @@ view: +subscriptions {
     hidden: yes
   }
 
+  dimension: is_ended {
+    description: "Indicates if subscription has ended"
+    type: yesno
+    sql: ${TABLE}.ended_at IS NOT NULL;;
+  }
+
+  dimension: plan_interval_type {
+    description: "Indicates the plan interval type (1 year, 6 month, 1 month, etc)"
+    type: string
+    sql: CONCAT(CAST(${TABLE}.plan_interval_count AS STRING),"_",  ${TABLE}.plan_interval);;
```

Comment:
the `CAST` here shouldn't be necessary, and fields should be referenced as `${field}` not `${TABLE}.field`:
```suggestion
    sql: CONCAT(${plan_interval_count}, "_",  ${plan_interval});;
```